### ISSUE TITLE
Laravel production error (8.27)

### ISSUE THREAD

**Issue by Abdullah-Hejazi** [laravel/framework#36223]
- Laravel Version: 8.27.0
- PHP Version: 7.4
- Database Driver & Version: mysql

### Description:
Laravel on production seems to have this bug that displays a lot of bugs, and all requests to laravel result in 500 error.
Log file with the messages related to this bug is bellow:
[laravel-2021-02-11.log](https://github.com/laravel/framework/files/5963318/laravel-2021-02-11.log)

BTW it seems to be working find on development (on windows), but not on production server (linux - ubuntu).

### Steps To Reproduce:
Using version: "version": "v8.27.0",
deploy on production on:
Ubuntu 18.04.3 LTS
PHP 7.4.14
Apache2 webserver

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by dkulyk** [comment]
See phpinfo() on production web server. Your PHP version is less than 7.3

**Comment by Abdullah-Hejazi** [comment]
> See phpinfo() on production web server. Your PHP version is less than 7.3

@dkulyk  My php version is 7.4.14, it should be compatible with laravel 8

**Comment by rs-sliske** [comment]
my guess would be apache is using a different version of php than the one you checked the version of on cli

**Comment by Abdullah-Hejazi** [comment]
@rs-sliske  php version that apache uses is also 7.4.14

I tried a lot of things, which is why I came here since I thought it might be a problem with the framework itself.

**Comment by Abdullah-Hejazi** [comment]
@driesvints  the reason I put it here is that the errors all seem to be inside the laravel/framework package rather than my code.

Take a look at the path here for example:
`
syntax error, unexpected ')' at /var/www/html/backend/staging/releases/backend_14/vendor/laravel/framework/src/Illuminate/Testing/Concerns/TestDatabases.php:151)
`

**Comment by dkulyk** [comment]
This is valid code for php >=7.3
```php
config()->set(
  "database.connections.{$default}.url",
  preg_replace('/^(.*)(\/[\w-]*)(\??.*)$/', "$1/{$database}$3", $url),
);
```

**Comment by Abdullah-Hejazi** [comment]
@dkulyk  I know, that was the strange part :(

Anyways I fixed the issue by upgrading to php 8.0 from php7.4

Thank you all
